src/db/migrations/20220528044344_processedArtworks.ts

```diff
@@ -0,0 +1,19 @@
+import { Knex } from 'knex';
+
+import { Table } from '../db';
+
+export const up = async (knex: Knex) => {
+  console.log('Running create contracts bootstrap');
+  await knex.schema.createTable(Table.processedArtworks, (table: Knex.CreateTableBuilder) => {
+    table.increments('id');
```

Comment:
hmm not sure if should have an increment id? trackId should be unique could just be used as the primary key. we do just use this in other places - don't think there are any autoincrementing ids anywhere else.

one of the motivation & requirements for no autoincrementing ids (although, this motivation/requirement may not be that important) is to ensure that if we open source this and someone else runs it with their own endpoint, their DB looks the same and has the same ids as ours. With autoincrementing ids, this can't be guaranteed, for example consider each of the below scenarios:
 - we run the db
 - we add sound, catalog
 - we process sound tracks + catalog tracks on tuesday
 - on tuesday process sound tracks+catalog tracks from tuesday and we add noizd and process noizd tracks from monday and tuesday
 - on weds we process tracks from all 3

2 weeks later, someone else runs the thing:
 - they process all sound+catalog+noizd tracks from tuesday and wednesday

the other person will end up with a different order of IDs, since our initial run had the monday noizd tracks mixed in.

the current design has a nice property across all tables where the order of all operations in the pipeline is basically irrelevant and 2 different people running it will eventually converge on to mostly the same state and same db contents (with some minor buggy exceptions probably)

so this also means that if we add a peer-to-peer network at some point in future where different nodes share and sync their data, it will be easy to do so if everyone is on the same page about ordering, as there will be no need for debate or consensus between nodes on ordering.

Reply:
ye i figured this would be a problem based on the other code that i read through :). was being a bit lazy. i cant use trackId since there are multiple images per track but i could follow your format that makes ids deterministic

Reply:
ye i mean it may make sense to actually process images from the nft directly rather than from the track, and use the nft id too. would need to handle the case where multiple nfts all have the same image tho.